history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskClaimAccTest.java

```diff
@@ -40,14 +43,52 @@ void should_CreateClaimedHistoryEvent_When_TaskIsClaimed() throws Exception {
 
     assertThat(taskService.getTask(taskId).getState()).isEqualTo(TaskState.READY);
     Task task = taskService.claim(taskId);
+
     assertThat(task.getState()).isEqualTo(TaskState.CLAIMED);
 
     events =
         taskHistoryQueryMapper.queryHistoryEvents(
             (TaskHistoryQueryImpl) historyService.createTaskHistoryQuery().taskIdIn(taskId));
 
-    String eventType = events.get(0).getEventType();
+    TaskHistoryEvent event = events.get(0);
+    String eventType = event.getEventType();
```

Comment:
why split?

Reply:
I deleted `String eventType` but did not delete TaskHistoryEvent event so that it's clear for others that events.get(0) is a TaskHistoryEvent.